src/server/pps/server/worker_rc.go

```diff
@@ -555,6 +561,42 @@ func (kd *kubeDriver) getStorageEnvVars(pipelineInfo *pps.PipelineInfo) []v1.Env
 			Value: strconv.FormatInt(kd.config.StorageCompactionShardCountThreshold, 10),
 		})
 	}
+	if kd.config.StorageMemoryThreshold > 0 {
+		vars = append(vars, v1.EnvVar{
+			Name:  StorageMemoryThresholdEnvVar,
+			Value: strconv.FormatInt(kd.config.StorageMemoryThreshold, 10),
+		})
+	}
+	if kd.config.StorageLevelFactor > 0 {
+		vars = append(vars, v1.EnvVar{
+			Name:  StorageLevelFactorEnvVar,
+			Value: strconv.FormatInt(kd.config.StorageLevelFactor, 10),
+		})
+	}
+	if kd.config.StorageCompactionMaxFanIn != 10 {
```

Comment:
I am checking the defaults here because that is the established pattern, but I could also just always append these values. Would be interested in hearing peoples thoughts on what is better.